src/final_grade.py

```diff
@@ -0,0 +1,14 @@
+def final_grade(exam, projects):
+    return (
+        100
+        if exam > 90 or projects > 10
+        else 90
+        if exam > 75 and projects >= 5
+        else 75
+        if exam > 50 and projects >= 2
+        else 0
```

Comment:
While this may work, it's not readable and not 'Pythonic'.

Change this to be if/elif statement with returns for each case that applies.

This syntax `100 if exam > 90 or projects > 10` is 100% ok when it makes the code more readable.  Like so.
```python
fruit_name = "apple" if fruit_is_red() else "orange"
```
The key is readability. One might be tempted to think `I can read this just fine`, but this certainly isn't the case. If it takes more than a split second to grok what is going on then there is a `code smell` that should be avoided.

Good job finding/writing something I haven't seen before, (though probably for good reason).

Reply:
https://www.codewars.com/kata/5ad0d8356165e63c140014d4/train/python

Reply:
I just want you to know that the program only looked like that after I ran it through black *-*

Reply:
I know, even if it were on a single line, it's unacceptable.  My example isn't fine because it's on a single line, but rather because it's short and easy to read.